CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 
 ## [Unreleased](https://github.com/idealista/prometheus_consul_exporter-role/tree/develop)
-
+### Added
+- Updagrading consul_exporter from [0.3.0](https://github.com/prometheus/consul_exporter/releases/tag/v0.3.0) to [0.7.1](https://github.com/prometheus/consul_exporter/releases/tag/v0.7.1)
+- Ansible version upgraded from 2.4.x to 2.9.x
+- Goss version upgraded from [0.3.5](https://github.com/aelsabbahy/goss/releases/tag/v0.3.5) to [0.3.13](https://github.com/aelsabbahy/goss/releases/tag/v0.3.13)
```

Comment:
It would be nice to add your GitHub user at the end of each line: https://github.com/idealista/prom2teams/blob/develop/CHANGELOG.md